source/_integrations/fritz.markdown

```diff
@@ -79,6 +79,8 @@ These can be changed at **AVM FRITZ!Box Tools** -> **Configure** on the Integrat
 
 ### Parental control
 
+Parental control switches can be used to enable and disable internet access of individual devices. If a device has internet access (no internet access) it will be enabled (disabled). You can also find the current blocking state of the individual devices in the UI of the FRITZ!Box under `Internet` -> `Filters` -> `Parental Controls` -> `Device Block`
```

Comment:
```suggestion
Parental control switches can be used to enable and disable internet access of individual devices. If a device has internet access it will be enabled, otherwise it will be disabled. You can also find the current blocking state of the individual devices in the UI of the FRITZ!Box under `Internet` -> `Filters` -> `Parental Controls` -> `Device Block`
```